docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -225,6 +225,18 @@ NOTE: Implementing xref:api/introspection.adoc#SRC5Component[SRC5Component] is a
 * xref:#ERC721-tokenURI[`++tokenURI(self, tokenId)++`]
 --
 
+[.contract-index#ERC721Component-Embeddable-Mixin-Impl]
+.Embeddable Mixin implementation
```

Comment:
```suggestion
.Embeddable Mixin
```
is it too much to link here to the _Mixins_ section of _Components_?

Reply:
Done!